static/js/dashboard-logs-results-grid.js

```diff
@@ -102,7 +102,7 @@ function getPanelGridOptions() {
                 if (diff <= 5) {
                     let scrollingTrigger = true;
                     data = getQueryParamsData(scrollingTrigger);
-                    runPanelLogsQuery(data);
+                    
```

Comment:
this is needed to have scrolling data. If you remove runPanelLogsQuery we will always get fixed set of records without scrolling

Reply:
okay, would modify the function then.